internal/pkg/service/buffer/storage/local/writer/test/benchmark/benchmark.go

```diff
@@ -0,0 +1,140 @@
+package benchmark
+
```

Comment:
Package `benchmark` obsahuje kod na testovanie prepustnosti zapisu tabulkovych dat na disk.

Dal som to ako samostatny pkg, aby to slo pouzit v buducnosti aj pre ine typy suborov.